enterprise/backend/src/metabase_enterprise/serialization/v2/storage/yaml.clj

```diff
@@ -0,0 +1,41 @@
+(ns metabase-enterprise.serialization.v2.storage.yaml
+  (:require [clojure.java.io :as io]
+            [metabase-enterprise.serialization.v2.storage :as storage]
+            [yaml.core :as yaml]))
+
+(defn- spit-yaml
+  [path obj]
+  (apply io/make-parents path)
+  (spit (apply io/file path) (yaml/generate-string obj :dumper-options {:flow-style :block})))
+
+(defn- store-entity! [{:keys [root-dir]} {{:keys [id type label]} :serdes/meta :as entity}]
+  (let [basename (if (nil? label)
+                   (str id ".yaml")
+                   ; + is a legal, unescaped character on all common filesystems, but not `identity-hash` or NanoID!
```

Comment:
nit: we often use `;;`

Reply:
Just to be clear, you mean to use `;;` for the comment, not instead of `+` in the filename?

Reply:
yes, I meant the comment thing